docs/paper/dev/api/event-api/chat-event.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/chat-events
+description: How to properly use the new AsyncChatEvent 
```

Comment:
personally not a big fan of this description

Reply:
```suggestion
description: An outline on the AsyncChatEvent and how to handle it
```